src/collapse/test/collapse.spec.js

```diff
@@ -1,55 +1,76 @@
-describe('collapse directive', function() {
+describe('collapse directive', function () {
```

Comment:
This whitespace change is unnecessary - this will need to all be changed back on merge.